_data/companies.yml

```
@@ -405,11 +405,11 @@
   last_update: 2020-03-09
 
 - name: Cloudflare
```

```suggestion
- name: Cloudflare[[1]](https://blog.cloudflare.com/cloudflare-during-the-coronavirus-emergency/)
```
This way the link gets shown on the web.